scripts/check_env_vars.js

```diff
@@ -0,0 +1,131 @@
+const fs = require('fs')
```

Comment:
Would be nice to look into creating a lib soon so we don't need to duplicate this everywhere.

Reply:
Yes I absolutely agree!